tidb/run

```diff
@@ -0,0 +1,6 @@
+lein run test \
```

Comment:
run.sh 

Btw, I prefer moving this to a scirpts directory and we can provide multi other test scripts for different cases.